crates/cast/bin/opts.rs

```diff
@@ -524,11 +525,11 @@ pub enum CastSubcommand {
         /// The block height to query at.
         ///
         /// Can also be the tags earliest, finalized, safe, latest, or pending.
-        #[arg(long, short = 'B')]
+        #[clap(long, short = 'B')]
```

Comment:
these should be `arg(`

Reply:
Yeah, run `cargo check --features clap/deprecated`